test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1657,6 +1657,22 @@ def skipTest(self, reason=None):
 
     skip_test = skipTest
 
+    def add_pbs_python_path_to_sys_path(self):
+        """
+        Add the path to the installed PBS Python modules located in the PBS
+        installation directory to the module search path if the path is not
+        already present.
+        """
+        for lib_dir in ['lib64', 'lib']:
+            pbs_python_path = os.path.join(self.server.pbs_conf['PBS_EXEC'],
+                lib_dir, 'python', 'altair')
+        if os.path.isdir(pbs_python_path):
```

Comment:
Did you mean to indent this if statement to be under the for loop body?

Reply:
Yes I did.  Thanks for catching that!  It's fixed in commit 5591dd0695e62133343861880db91042ebc47ffa.